src/adapterManager.js

```diff
@@ -271,7 +271,7 @@ adapterManager.makeBidRequests = function(adUnits, auctionStart, auctionId, cbTi
   if (gdprDataHandler.getConsentData()) {
     bidRequests.forEach(bidRequest => {
       bidRequest['gdprConsent'] = gdprDataHandler.getConsentData();
-      bidRequest['ccpaConsent'] = uspDataHandler.getConsentData();
+      bidRequest['uspConsent'] = uspDataHandler.getConsentData();
```

Comment:
This is fine, but the naming convention seems to assume we will never need to fetch another consent or additional information via the `USP` framework.

`USP` consent is the framework while the consent data here is specific to `CCPA`, as `GDPR` is the consent fetched from what we defined as the `IAB` framework.

Reply:
not a bad point, but I think it's unclear.  The "command" we're retrieving the data from is "getUSPData", so I think calling it "usp" is fine.  perhaps in the future other types of consent would be included in the "uspData" or "uspString"